docs/source/author/writing_guide.rst

```diff
@@ -0,0 +1,439 @@
+Writing guide
+##############################################################################
+
+The majority of our text content is written in Markdown, and we also developed
+a program called Kordac to allow you to include HTML elements like images and
+videos with simple text tags.
+
+For example, the following text:
+
+.. code-block:: none
+
+  Try using the following blocks:
+
+  ```scratch
+  when flag clicked
+
+  say [Hello] for [2] secs
+  ```
+
+  **Tips**
+
+  - Make sure all your blocks are “snapped” together in a line like a jigsaw puzzle.
+
+will display as the following on the website:
+
+.. image:: ../_static/img/markdown_example_rendered.png
+  :alt: An image showing the above Markdown syntax rendered as HTML
+
+.. note::
+
+  If you already know Markdown syntax, please remember the following project
+  preferences (for consistency and readability):
+
+  - Use asterisks (``*``) for emphasis, instead or underscores.
+  - Use minuses (``-``) for unordered lists.
+  - No HTML within text files, we use Kordac text tags to add iframes,
+    images, videos, etc.
+
+Below is a basic guide to syntax for Markdown and Kordac text tags.
+When viewing Kordac documentation for a tag, the top of the page will detail
+how to use the tag in a basic example.
+Some text tags also have required and/or optional tag parameters for further
+configuration.
+
+.. contents:: Text Syntax
```

Comment:
maybe these should be split into normal markdown tags and kordac tags? I get that you've done it alphabetically, but I think it would be easier to read/follow if they were sorted into the two groups - then the kordac group could be given as a list of links to their processor docs, rather than heading/link/heading/link/heading/...